### ISSUE TITLE
Paper-print tiled maps in high-res

### ISSUE THREAD

**Issue by benbucksch** [openlayers/ol2#561]
I need to print a map on paper, with layers. I created a separate HTML page for that. However, the OSM (OpenStreetMap) base layer prints with a screen resolution of 72dpi, which means: 1) most street names are not visible (and on paper, that's final, no zooming) 2) those streets names that are printed are often very pixelish, almost unreadable.

This is because browsers, at least Mozilla, always print with 72dpi, to make webpages look on paper then same as on screen. But it is possible to provide higher-resolution images: <img style="width: 100px; height: 100px" src="img1000x1000px.png">, with "img1000x1000px.png" having 1000x1000px size, will give a high-res print in Mozilla.
This will print 1000 px on 3,55 cm on paper (100 pixels / 72 dots per inch \* 2,56 cm/inch = 3,55 cm), i.e. result in 720 dpi (1000px/100px \* 72 dpi).
In other words, the image file must have 10 times the pixel size than the size in page per CSS width/height.

This means, we need to adapt the OSM -> XYZ -> Grid -> Layer class for that. The <img> grid must be 10 times finer than it is for the screen.

**Comment by poussinxt** [comment]
I have the same problem
Did anybody find a solution?
